settings/Core.setting.php

```diff
@@ -239,6 +239,27 @@
     'description' => NULL,
     'help_text' => NULL,
   ],
+  'defaultExternUrl' => [
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'defaultExternUrl',
+    'type' => 'String',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => [
+      'class' => 'crm-select2',
+    ],
+    'default' => 'router',
+    'add' => '5.27',
+    'title' => ts('Extern URL Style'),
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'help_text' => NULL,
```

Comment:
Also - the help text should ideally be in here - although I think that if the settings page is not using the generic tpl it may not work

Reply:
That sounds cool :) - I didn't realize that dataflow was now working for some forms.

FWIW, when I grep `settings/` for other examples of `help_text`, they all appear to be a couple sentences long. Since this is actually several paragraphs (which in turn requires markup), perhaps it's still OK in the `.hlp` file?